input/resources/au-core-client.xml

```diff
@@ -216,42 +216,6 @@
 </tr>
```

Comment:
Fix "http://hl7.org/fhir/R4/composition.html#search" to be "http://hl7.org/fhir/R4/medicationrequest.html#search" - location of this comment is not related to where this fix needs to be applied

Reply:
Change applied (not pushed yet).